cypress/integration/test-helper.js

```diff
@@ -16,9 +16,14 @@ const shouldContainStyles = (element, css, styling) => {
   });
 };
 
+const shouldHaveMetadata = (element, content) => {
```

Comment:
This seems to be checking for the existence of an attribute on an element and therefore isn't jsut limited to metadata, if my thinking is correct could this be renamed to `shouldHaveAttribute`?